docker-compose-pool.yaml

```diff
@@ -0,0 +1,151 @@
+# Copyright 2019-2020 Intel Corporation
```

Comment:
Need to follow the approach in https://github.com/hyperledger/avalon/commit/1ba5385f33fe510be942d1675392659fb1e4bb99